unix.md

```diff
@@ -0,0 +1,13 @@
+# UNIX
+
+A multitasking, multi-user computer operating system originally developed in 1969, variants of which now form the underpinning of many operating systems, including OS X, iOS, and Linux.
+
+## Beginning UNIX
+
+* Read [The Linux Programming Interface](http://amzn.to/the-linux-programming-interface). Chapters 1,  2, 6, 8, 10, 14, 15, 18, 25, 44, 62.
+* Read [Learning the Shell](http://www.linuxcommand.org/lc3_learning_the_shell.php)
+* Read about [I/O redirection](http://en.wikipedia.org/wiki/Redirection_(computing)).
+
+### You should be able to
+
+* Use these commands and operators: `|`, `<`, `>`, `>>`, `&`, `ack`, `awk`, `cat`, `chmod`, `chown`, `cp`, `export`, `find`, `kill`, `locate`, `ls`, `mkdir`, `mv`, `ps`, `rm`, `sed`, `sort`, `tail`, `top`, `vim`, `whereis`, `xargs`.
```

Comment:
All of these are available out of the box on most systems except `ack` (and we would use `ag` these days anyway).